app/views/admins/dashboard/_counter.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="<%= col_class %>">
   <div class="card">
-    <div class="card-body p-2 text-center">
+    <div class="card-body p-2 text-center ">
```

Comment:
Empty space not necessary, pain attention on unnecessary modifications. It can mess up the code.